index.d.ts

```diff
@@ -1,9 +1,9 @@
-import { parsed } from "./lib/shared";
+import type { Pep440Version } from "./lib/shared";
 import type { SemVer } from "semver";
 
```

Comment:
```suggestion

export type { Pep440Constraint, Pep440Version };

```
For easier type usage in renovate.